python/tox.ini

```diff
@@ -50,9 +55,8 @@ commands =
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}
-passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*
+passenv = TOXENV CI CODECOV_*
 deps =
     -r{toxinidir}/requirements_dev.txt
 commands =
-    pip install -U pip
```

Comment:
we pin pip in `requirements_dev.txt` so universally upgrading here made no sense imo